spec/classes/mod/security_spec.rb

```diff
@@ -24,13 +24,20 @@
             )
           }
           it { is_expected.to contain_package('mod_security_crs') }
-          if facts[:os]['release']['major'].to_i > 6
+          if facts[:os]['release']['major'].to_i > 6 && facts[:os]['release']['major'].to_i <= 7
```

Comment:
we can do a compare using the gem version functionality. making it a single boolean check
``` if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')```